src/Service/ZaakTypeService.php

```diff
@@ -191,6 +191,23 @@ public function getZaakType(string $caseTypeID)
     }//end getZaakType()
 
 
+    /**
+     * Maps a simple informatieobjecttype.
+     *
+     * @param array $field xxllnc field.
+     *
+     * @return array InformatieObjectType.
+     */
+    private function mapInformatieObjectType(array $field): array
+    {
+        return [
+            'omschrijving'                => ($field['original_label'] ?? $field['label'] ?? $field['magic_string']),
```

Comment:
Codacy gaat dit niet leuk vinden, misschien gewoon if statements gebruiken aangezien deze functie toch nog klein is? 🤔